contact-form.css

```diff
@@ -0,0 +1,179 @@
+.contact-form {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  padding: 120px 24px 20px 24px;
+}
+
+.view {
+  color: #172b4d;
+  font-family: 'Inter', sans-serif;
+  font-size: 32px;
+  font-weight: 700;
+  font-style: normal;
+  line-height: 44px;
+  line-height: 114%;
+  text-align: center;
+}
+
+.touch {
+  width: 131px;
+  height: 48px;
+  background-color: #ff6b00;
+  margin: 30px 90px 100px 90px;
+  text-align: center;
+  border: transparent;
+  color: #fff;
+  font-weight: 700;
+}
+
+.touch:hover {
+  background: #fff5e1;
+  border: 1px solid #f55800;
+  color: #f55800;
+}
+
+.touch:active {
+  background: #e05304;
+  color: #f4f1f0;
+}
+
+.footer {
+  border-color: black;
+}
+
+.footer > .contact {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  gap: 16px;
+  list-style: none;
+}
+
+.contact > li > a {
+  width: 36px;
+  height: 36px;
+  color: #505f79;
+}
+
+.web > a :hover {
+  color: blue;
+}
+
+.line-three {
+  width: 125%;
+  border: 1px solid #172b4d;
+  align-self: flex-start;
+  margin-top: 25px;
+  margin-left: -24px;
+}
```

Comment:
- [ ] Kindly fix this line. It's causing problems with the width of your page.
<img width="938" alt="image" src="https://user-images.githubusercontent.com/77190203/150614789-0a5952a2-d3f3-4d26-a601-6d8d62c2ccc5.png">